SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -4,5 +4,5 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
erface <br> python2 <br> python3 <br> pytz <br> PyYAML <br> rapidjson <br> readline <br> redis <br> rng-tools <br> rpcbind <br> rpcsvc-proto <br> rpm <br> rpm-ostree <br> rrdtool <br> rsync <br> rsyslog <br> ruby <br> rubygem-bundler <br> rubygem-addressable <br> rubygem-async <br> rubygem-async-http <br> rubygem-async-io <br> rubygem-async-pool <br> rubygem-aws-eventstream <br> rubygem-aws-partitions <br> rubygem-aws-sdk-core <br> rubygem-aws-sdk-kms <br> rubygem-aws-sdk-s3 <br> rubygem-aws-sdk-sqs <br> rubygem-aws-sigv4 <br> rubygem-concurrent-ruby <br> rubygem-console <br> rubygem-cool.io <br> rubygem-digest-crc <br> rubygem-elasticsearch <br> rubygem-elasticsearch-api <br> rubygem-elasticsearch-transport <br> rubygem-excon <br> rubygem-faraday <br> rubygem-ffi <br> rubygem-fiber-local <br> rubygem-fluent-config-regexp-type <br> rubygem-fluent-logger <br> rubygem-fluent-plugin-elasticsearch <br> rubygem-fluent-plugin-kafka <br> rubygem-fluent-plugin-prometheus <br> rubygem-fluent-plugin-prometheus_pushgateway <br> rubygem-fluent-plugin-record-modifier <br> rubygem-fluent-plugin-rewrite-tag-filter <br> rubygem-fluent-plugin-s3 <br> rubygem-fluent-plugin-systemd <br> rubygem-fluent-plugin-td <br> rubygem-fluent-plugin-webhdfs <br> rubygem-fluentd <br> rubygem-hirb <br> rubygem-http_parser.rb <br> rubygem-httpclient <br> rubygem-jmespath <br> rubygem-ltsv <br> rubygem-mini_portile2 <br> rubygem-msgpack <br> rubygem-multi_json <br> rubygem-multipart-post <br> rubygem-nio4r <br> rubygem-nokogiri <br> rubygem-oj <br> rubygem-parallel <br> rubygem-prometheus-client <br> rubygem-protocol-hpack <br> rubygem-protocol-http <br> rubygem-protocol-http1 <br> rubygem-protocol-http2 <br> rubygem-public_suffix <br> rubygem-quantile <br> rubygem-rake <br> rubygem-rdkafka <br> rubygem-ruby-kafka <br> rubygem-ruby-progressbar <br> rubygem-ruby2_keywords <br> rubygem-rubyzip <br> rubygem-serverengine <br> rubygem-sigdump <br> rubygem-strptime <br> rubygem-systemd-journal <br> rubygem-td <br> rubygem-td-client <br> rubygem-td-logger <br> rubygem-timers <br> rubygem-tzinfo <br> rubygem-tzinfo-data <br> rubygem-webhdfs <br> rubygem-yajl-ruby <br> rubygem-zip-zip <br> runc <br> rust <br> scons <br> sed <br> sg3_utils <br> shadow-utils <br> slang <br> snappy <br> socat <br> sqlite <br> sshpass <br> strace <br> strongswan <br> subversion <br> sudo <br> swig <br> syslinux <br> syslog-ng <br> sysstat <br> systemd <br> systemtap <br> tar <br> tboot <br> tcl <br> tcpdump <br> tcp_wrappers <br> tcsh <br> td-aget <br> tdnf <br> telegraf <br> texinfo <br> tmux <br> tpm2-abrmd <br> tpm2-tools <br> tpm2-tss <br> traceroute <br> tree <br> trousers <br> tzdata <br> unbound <br> unixODBC <br> unzip <br> usbutils <br> userspace-rcu <br> utf8proc <br> util-linux <br> valgrind <br> vim <br> vsftpd <br> WALinuxAgent <br> wget <br> which <br> wpa_supplicant <br> xerces-c <br> xfsprogs <br> xinetd <br> xmlsec1 <br> xmlto <br> xz <br> zchunk <br> zeromq <br> zip <br> zlib <br> zsh <br>  |
```

Comment:
><br> rubygem-bundler <br> rubygem-addressable <br> rubygem-async <br> rubygem-async-http <br> rubygem-async-io <br> rubygem-async-pool <br> rubygem-aws-eventstream <br> rubygem-aws-partitions <br> rubygem-aws-sdk-core <br> rubygem-aws-sdk-kms <br> rubygem-aws-sdk-s3 <br> rubygem-aws-sdk-sqs <br> rubygem-aws-sigv4 <br> rubygem-concurrent-ruby <br> rubygem-console <br> rubygem-cool.io <br> rubygem-digest-crc <br> rubygem-elasticsearch <br> rubygem-elasticsearch-api <br> rubygem-elasticsearch-transport <br> rubygem-excon <br> rubygem-faraday <br> rubygem-ffi <br> rubygem-fiber-local <br> rubygem-fluent-config-regexp-type <br> rubygem-fluent-logger <br> rubygem-fluent-plugin-elasticsearch <br> rubygem-fluent-plugin-kafka <br> rubygem-fluent-plugin-prometheus <br> rubygem-fluent-plugin-prometheus_pushgateway <br> rubygem-fluent-plugin-record-modifier <br> rubygem-fluent-plugin-rewrite-tag-filter <br> rubygem-fluent-plugin-s3 <br> rubygem-fluent-plugin-systemd <br> rubygem-fluent-plugin-td <br> rubygem-fluent-plugin-webhdfs <br> rubygem-fluentd <br> rubygem-hirb <br> rubygem-http_parser.rb <br> rubygem-httpclient <br> rubygem-jmespath <br> rubygem-ltsv <br> rubygem-mini_portile2 <br> rubygem-msgpack <br> rubygem-multi_json <br> rubygem-multipart-post <br> rubygem-nio4r <br> rubygem-nokogiri <br> rubygem-oj <br> rubygem-parallel <br> rubygem-prometheus-client <br> rubygem-protocol-hpack <br> rubygem-protocol-http <br> rubygem-protocol-http1 <br> rubygem-protocol-http2 <br> rubygem-public_suffix <br> rubygem-quantile <br> rubygem-rake <br> rubygem-rdkafka <br> rubygem-ruby-kafka <br> rubygem-ruby-progressbar <br> rubygem-ruby2_keywords <br> rubygem-rubyzip <br> rubygem-serverengine <br> rubygem-sigdump <br> rubygem-strptime <br> rubygem-systemd-journal <br> rubygem-td <br> rubygem-td-client <br> rubygem-td-logger <br> rubygem-timers <br> rubygem-tzinfo <br> rubygem-tzinfo-data <br> rubygem-webhdfs <br> rubygem-yajl-ruby <br> rubygem-zip-zip [](start = 7264, length = 1966)

Since all of these SPEC files were created by us, then they are MIT licensed by Microsoft and are not from Photon.  Only tag SPEC files under "Photon" that actually originated from Photon.

Reply:
Fixed.

td-agent and librdkafka are imported from new sources that did not appear in Mariner before. I'm using Openmamba as the example to create the two new instances.